web/frontend/src/components/slam.vue

```diff
@@ -508,7 +508,7 @@ onUnmounted(() => {
           />
         </div>
       </div>
-      <div class="gap-4x border-border-1 justify-start sm:border-l">
+      <div class="gap-4x border-border-1 w-full justify-start sm:border-l">
```

Comment:
this is all that we needed 🤯 such that we have proper size on a larger computer monitor.

Reply:
🎉